src/dev/arm/smmu_v3.cc

```diff
@@ -310,7 +310,8 @@ SMMUv3::runProcessTiming(SMMUProcess *proc, PacketPtr pkt)
             DPRINTF(SMMUv3, "[t] master req  addr=%#x size=%#x\n",
                     action.pkt->getAddr(), action.pkt->getSize());
 
-            if (packetsToRetry.empty() && masterPort.sendTimingReq(action.pkt)) {
+            if (packetsToRetry.empty() &&
+                    masterPort.sendTimingReq(action.pkt)) {
```

Comment:
There's nothing wrong with this change :). However, since we are no longer changing the masterPort name here, we should revert this to the original.